src/electronic-ids/pcsc/FinEID.hpp

```diff
@@ -32,8 +32,11 @@ class FinEIDv3 : public PcscElectronicID
 public:
     FinEIDv3(pcsc_cpp::SmartCard::ptr _card) : PcscElectronicID(std::move(_card)) {}
 
-private:
-    pcsc_cpp::byte_vector getCertificateImpl(const CertificateType type) const override;
+protected:
+    using byte_vector = pcsc_cpp::byte_vector;
+    using value_type = byte_vector::value_type;
```

Comment:
`FinEIDv3::value_type` reads "the value type of FinEIDv3". In my opinion "the byte type of FinEIDv3" is more correct:
```suggestion
    using byte_type = byte_vector::value_type;
```

Reply:
It is just alias tho shorten. Please define byte type in pcsc-cpp

Reply:
Will do, thanks for the suggestion!